src/pypistats/__init__.py

```diff
@@ -261,7 +264,7 @@ def _date_range(data: dict | list) -> tuple[str | None, str | None]:
     try:
         first = data[0]["date"]
         last = data[0]["date"]
-    except KeyError:
+    except (KeyError, IndexError):
```

Comment:
Is this no longer needed?
```suggestion
    except KeyError:
```

Reply:
Yep, I'll remove it.

Reply:
Done